src/login/logind-button.c

```diff
@@ -15,7 +15,40 @@
 #include "missing_input.h"
 #include "string-util.h"
 
-#define CONST_MAX5(a, b, c, d, e) CONST_MAX(CONST_MAX(a, b), CONST_MAX(CONST_MAX(c, d), e))
+/* KEY_RESTART is the highest value key in keys_interested. */
+#define _KEY_MAX_INTERESTED KEY_RESTART
+
+/* Adding more values here may require _KEY_MAX_INTERESTED to be updated, this array must be sorted by key value. */
+static const int keys_interested[] = {
+        KEY_ESC,
+        KEY_LEFTCTRL,
+        KEY_LEFTSHIFT,
+        KEY_RIGHTSHIFT,
+        KEY_LEFTALT,
+        KEY_RIGHTCTRL,
+        KEY_RIGHTALT,
+        KEY_POWER,
+        KEY_LEFTMETA,
+        KEY_RIGHTMETA,
```

Comment:
why meta? we don't actually use that, do we?

Reply:
No, it's not used, but I put it there for what I felt was completeness (in case someone wants to add a binding in the future).

Reply:
please drop. we really don't want to track more stuff thatn we really need.

Reply:
ok